src/Me/MentorshipReq/MentorshipReq.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {
+  fireEvent,
+  act,
+  render,
+  cleanup,
+  reqData,
+  screen,
+  waitForElementToBeRemoved,
+  userData,
+} from './test-setup';
+import MentorshipReq from './MentorshipReq';
+import { STATUS } from '../../helpers/mentorship';
+import * as api from '../../api';
+
+function mockMentorshipApi({ getReq, updateStatus }) {
+  api.getMentorshipRequests = jest.fn(() => Promise.resolve(getReq));
+  api.updateMentorshipReqStatus = jest.fn(() =>
+    Promise.resolve({
+      success: updateStatus?.success,
+      mentorship: { status: updateStatus.status ?? STATUS.viewed },
+    })
+  );
+}
+
+describe('MentorshipReq', () => {
+  jest.mock('../../api/index.js');
+  beforeEach(() => {
+    api.getCurrentUser = jest.fn(() => Promise.resolve(userData));
+  });
+
+  describe('MentorshipReq List', () => {
+    afterEach(cleanup);
+
+    it('Fetch and render a list of mentorship requests', async () => {
+      mockMentorshipApi({ getReq: reqData.data });
+      const { findAllByText } = render(<MentorshipReq />);
+
+      //FIX: await waitForElementToBeRemoved(() => screen.getByRole('status'));
```

Comment:
?

Reply:
Remainder to look into fixing this line

Reply:
Can you create an issue for this? We can even add the issue number / link here.